### PR TITLE
Chapter1/homework/1 and2 - Homework Assignment

### PR DESCRIPTION
**Pull request by Joshhortt**
1. Print to console all the info based on the example requested, using **concatenation**. 
    At first I declared some variables type: **double**, **int,** and **string**.

2. Body Mass Index Calculation is based on weight and height.  (BMI)=weight/(height*height), *10000
    where weight is in kilograms and height in centimeters. The program takes weight and height as 
    input and displays BMI using a method called **Math.Round** with an value to the specified 
    number of **decimal digits of 2**.

3. Print to console all the personal details requested based on any **users input**, using the method 
    **Convert.ToInt32** to convert specified values to signed integers. 
    Use of **concatenation** to print results to the console. Declared some type **double** variables.

3. Print to console all the personal details requested for Body Mass Index using the same calculations 
    in point number 2, but with request of **user input** using the method **double.Parse** that 
    converts the string representation of a number, to its double-precision floating-point number.

### CODE REVIEW

File: Src/BootCamp.Chapter/Program.cs [hunk 1]
```diff
@@ -1,12 +1,80 @@
...
+			   his weight is 50 kg and his height is 156.5 cm.
+			 */
+
+			double height = 156.5;
+			int weight = 50;
+			int age = 19;
+			string surName = "Jefferson";
+			string name = "Tom";
+
+			Console.WriteLine(name + " " + surName + " you are " + age + " years old, " +
```
Comment (HellHunterMax) [inline Src/BootCamp.Chapter/Program.cs]: When making a string a small tip would be to add $ i front of the string that way you dont have to + all the time or use numbers like you did later on.
so: Console.WriteLine($"{name} {surName} you are {age} years old, your weight is {weight} Kg and your height is {height} cm")

File: Src/BootCamp.Chapter/Program.cs [hunk 2]
```diff
@@ -1,12 +1,80 @@
...
+			 1.Print all the info based on the example message below:
+			   Tom Jefferson is 19 years old, 
+			   his weight is 50 kg and his height is 156.5 cm.
+			 */
+
+			double height = 156.5;
+			int weight = 50;
+			int age = 19;
+			string surName = "Jefferson";
+			string name = "Tom";
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs:24]: Read input from console.

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -1,12 +1,80 @@
...
+
+			Console.Write("Enter your height in centimeters: ");
+			centimeters = double.Parse(Console.ReadLine());
+
+			Console.Write("Enter your weight in kilograms: ");
+			kilograms = double.Parse(Console.ReadLine());
+
+			BMI = kilograms / (centimeters * centimeters) * 10000;
+			Console.WriteLine("Your Body Mass index 'BMI' is: {0}", Math.Round(bmi, 2));
+			Console.Write("\n");
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Using writeline instead is more simple.

**Review by Almantask — CHANGES_REQUESTED** [review]

File: Src/BootCamp.Chapter/Program.cs (hunk 1 above)
Comment (Joshhortt) [reply Src/BootCamp.Chapter/Program.cs]: Thanks Max for the great Tip. Just changed those Lines. 
Mission Done!

**Pull request closed without merging**